--- tamil-part.lexc
+++ tamil-part.lexc
@@ -1,7 +1,7 @@
 !!!part.lexc!!!
 ! By K. Sarveswaran, iamsarves at gmail dot com.
 
-Multichar_Symbols  +intensifier +sandhik +sandhic +sandhip +sandhit +pron +quant +all +human +inter +poss +det +degree +same +distrib +modal +neg +exist +ques +prohib +abbrev +letter
+Multichar_Symbols  +intensifier +sandhik +sandhic +sandhip +sandhit +pron +quant +all +human +inter +poss +det +degree +same +distrib +modal +neg +exist +ques +prohib +abbrev +letter +acc
 
 LEXICON Root
 abbreviations;
@@ -84,6 +84,8 @@
 என்று  complimentiser;
 என  complimentiser;
 என்பது  complimentiser;
+என்பதை+comp+acc:என்பதை #;
+எனக்+comp+sandhik:எனக் #;
 
 LEXICON intensifiers
 மிக		intensifier;
