--- ThamizhiVerbs-otherthan-3-4-62-11-12.lexc
+++ ThamizhiVerbs-otherthan-3-4-62-11-12.lexc
@@ -80760,6 +80760,7 @@
 அங்குரிக்கப்படு                   CPadu61Vinf; 
 அங்கலாய்க்கப்படு                   CPadu61Vinf; 
 அங்கீகரிக்கப்படு                   CPadu61Vinf; 
+பயன்படுத்தப்படு                   CPadu61Vinf; 
 அஞ்சிக்கப்படு                   CPadu61Vinf; 
 அஞ்சலிக்கப்படு                   CPadu61Vinf; 
 வழிக்கப்படு                   CPadu61Vinf; 
