--- ThamizhiVerbs-C4.lexc
+++ ThamizhiVerbs-C4.lexc
@@ -475,6 +475,7 @@
 +verb+fin+sim+strong+pres%=கின்ற்+euph%=அன்+1sg%=அன்:^கின்றனன் #;
 +verb+fin+sim+strong+pres%=கின்ற்+euph%=அன்+3sgm%=அன்:^கின்றனன் #;
 +verb+fin+sim+strong+pres%=கின்ற்+euph%=அன்+3sgf%=அள்:^கின்றனள் #;
++verb+nonfin+sim+verbalnoun%=தல்:^தல் #;
 +verb+fin+sim+imp%=∅+2sg%=∅:^ #;
 +verb+fin+sim+imp%=∅+2pl%=உங்கள்:^ுங்கள் #;
 +verb+fin+sim+imp%=∅+neg%=ஆத்+euph%=ஏ:^ாதே #;
