--- ThamizhiVerbs-otherthan-3-4-62-11-12.lexc
+++ ThamizhiVerbs-otherthan-3-4-62-11-12.lexc
@@ -121,6 +121,7 @@
 +verb+fin+sim+strong+pres%=கின்ற்+euph%=அன்+1sg%=அன்:^கின்றனன் #;
 +verb+fin+sim+strong+pres%=கின்ற்+euph%=அன்+3sgm%=அன்:^கின்றனன் #;
 +verb+fin+sim+strong+pres%=கின்ற்+euph%=அன்+3sgf%=அள்:^கின்றனள் #;
++verb+nonfin+sim+verbalnoun%=தல்:^தல் #;
 +verb+fin+sim+imp%=∅+2sg%=∅:^ #;
 +verb+fin+sim+imp%=∅+2pl%=உங்கள்:^ுங்கள் #;
 +verb+fin+sim+imp%=∅+neg%=ஆத்+euph%=ஏ:^ாதே #;
@@ -11684,9 +11685,10 @@
 தொடு      C6Vinf;
 வீறிடு      C6Vinf;
 போடு    C6Vinf;
 
 
 LEXICON C6Vinf
++verb+nonfin+sim+verbalnoun%=தல்:^தல் #;
 
 
 
