--- ThamizhiVerbs-otherthan-3-4-62-11-12.lexc
+++ ThamizhiVerbs-otherthan-3-4-62-11-12.lexc
@@ -37,6 +37,7 @@
 ClassPoModern;
 ClassVaaModern;
 ClassKelModern;
+ClassNilModern;
 ClassUlExistential;
 
 
@@ -11628,6 +11629,106 @@
 +verb+nonfin+sim+pres%=கின்ற்+adjpart%=அ:^கின்ற #;
 +verb+nonfin+sim+pres%=கிற்+adjpart%=அ:^கிற #;
 +verb+nonfin+sim+futANDadjpart%=உம்:^ம் #;
+
+LEXICON ClassNilModern
+
+நில்:நில் NilBaseVinf;
+நில்:நின் NilPastVinf;
+நில்:நிற் NilNonpastVinf;
+
+LEXICON NilPastVinf
++verb+fin+sim+strong+past%=ற்+1sg%=ஏன்:^றேன் #;
++verb+fin+sim+strong+past%=ற்+1pl%=ஓம்:^றோம் #;
++verb+fin+sim+strong+past%=ற்+2sg%=ஆய்:^றாய் #;
++verb+fin+sim+strong+past%=ற்+2sgh%=ஈர்:^றீர் #;
++verb+fin+sim+strong+past%=ற்+2pl%=ஈர்கள்:^றீர்கள் #;
++verb+fin+sim+strong+past%=ற்+3sgm%=ஆன்:^றான் #;
++verb+fin+sim+strong+past%=ற்+3sgf%=ஆள்:^றாள் #;
++verb+fin+sim+strong+past%=ற்+3sge%=அர்:^றர் #;
++verb+fin+sim+strong+past%=ற்+3ple%=னர்:^றனர் #;
++verb+fin+sim+strong+past%=ற்+3ple%=ஆர்கள்:^றார்கள் #;
++verb+fin+sim+strong+past%=ற்+3sghe%=ஆர்:^றார் #;
++verb+fin+sim+strong+past%=ற்+3sghe%=ஆர்கள்:^றார்கள் #;
++verb+fin+sim+strong+past%=ற்+3sgn%=அது:^றது #;
++verb+fin+sim+strong+past%=ற்+3pln%=அன:^றன #;
++verb+nonfin+sim+vpart%=உ:^று #;
++verb+nonfin+sim+vpart%=உ+sandhic%=ச்:^றுச் #;
++verb+nonfin+sim+vpart%=உ+sandhit%=த்:^றுத் #;
++verb+nonfin+sim+vpart%=உ+sandhik%=க்:^றுக் #;
++verb+nonfin+sim+vpart%=உ+sandhip%=ப்:^றுப் #;
++verb+nonfin+sim+con%=ஆல்:^றால் #;
++verb+nonfin+sim+past%=ற்+adjpart%=அ:^ற #;
+
+LEXICON NilNonpastVinf
++verb+fin+sim+strong+pres%=கிற்+1sg%=ஏன்:^கிறேன் #;
++verb+fin+sim+strong+pres%=கிற்+1pl%=ஓம்:^கிறோம் #;
++verb+fin+sim+strong+pres%=கிற்+2sg%=ஆய்:^கிறாய் #;
++verb+fin+sim+strong+pres%=கிற்+2sgh%=ஈர்:^கிறீர் #;
++verb+fin+sim+strong+pres%=கிற்+2pl%=ஈர்கள்:^கிறீர்கள் #;
++verb+fin+sim+strong+pres%=கிற்+3sgm%=ஆன்:^கிறான் #;
++verb+fin+sim+strong+pres%=கிற்+3sgf%=ஆள்:^கிறாள் #;
++verb+fin+sim+strong+pres%=கிற்+3sge%=அர்:^கிறர் #;
++verb+fin+sim+strong+pres%=கிற்+3ple%=னர்:^கிறனர் #;
++verb+fin+sim+strong+pres%=கிற்+3ple%=ஆர்கள்:^கிறார்கள் #;
++verb+fin+sim+strong+pres%=கிற்+3sghe%=ஆர்:^கிறார் #;
++verb+fin+sim+strong+pres%=கிற்+3sghe%=ஆர்கள்:^கிறார்கள் #;
++verb+fin+sim+strong+pres%=கிற்+3sgn%=அது:^கிறது #;
++verb+fin+sim+strong+pres%=கிற்+3pln%=அன:^கிறன #;
++verb+fin+sim+strong+pres%=கின்ற்+1sg%=ஏன்:^கின்றேன் #;
++verb+fin+sim+strong+pres%=கின்ற்+1pl%=ஓம்:^கின்றோம் #;
++verb+fin+sim+strong+pres%=கின்ற்+2sg%=ஆய்:^கின்றாய் #;
++verb+fin+sim+strong+pres%=கின்ற்+2sgh%=ஈர்:^கின்றீர் #;
++verb+fin+sim+strong+pres%=கின்ற்+2pl%=ஈர்கள்:^கின்றீர்கள் #;
++verb+fin+sim+strong+pres%=கின்ற்+3sgm%=ஆன்:^கின்றான் #;
++verb+fin+sim+strong+pres%=கின்ற்+3sgf%=ஆள்:^கின்றாள் #;
++verb+fin+sim+strong+pres%=கின்ற்+3sge%=அர்:^கின்றர் #;
++verb+fin+sim+strong+pres%=கின்ற்+3ple%=னர்:^கின்றனர் #;
++verb+fin+sim+strong+pres%=கின்ற்+3ple%=ஆர்கள்:^கின்றார்கள் #;
++verb+fin+sim+strong+pres%=கின்ற்+3sghe%=ஆர்:^கின்றார் #;
++verb+fin+sim+strong+pres%=கின்ற்+3sghe%=ஆர்கள்:^கின்றார்கள் #;
++verb+fin+sim+strong+pres%=கின்ற்+3sgn%=அது:^கின்றது #;
++verb+fin+sim+strong+pres%=கின்ற்+3pln%=அன:^கின்றன #;
++verb+fin+sim+middle+fut%=ப்+1sg%=ஏன்:^பேன் #;
++verb+fin+sim+middle+fut%=ப்+1pl%=ஓம்:^போம் #;
++verb+fin+sim+middle+fut%=ப்+2sg%=ஆய்:^பாய் #;
++verb+fin+sim+middle+fut%=ப்+2sgh%=ஈர்:^பீர் #;
++verb+fin+sim+middle+fut%=ப்+2pl%=ஈர்கள்:^பீர்கள் #;
++verb+fin+sim+middle+fut%=ப்+3sgm%=ஆன்:^பான் #;
++verb+fin+sim+middle+fut%=ப்+3sgf%=ஆள்:^பாள் #;
++verb+fin+sim+middle+fut%=ப்+3sge%=அர்:^பர் #;
++verb+fin+sim+middle+fut%=ப்+3ple%=னர்:^பனர் #;
++verb+fin+sim+middle+fut%=ப்+3ple%=ஆர்கள்:^பார்கள் #;
++verb+fin+sim+middle+fut%=ப்+3sghe%=ஆர்:^பார் #;
++verb+fin+sim+middle+fut%=ப்+3sghe%=ஆர்கள்:^பார்கள் #;
++verb+fin+sim+strong+fut%=உம்+3sgn%=∅:^கும் #;
++verb+fin+sim+strong+fut%=உம்+3pln%=∅:^கும் #;
++verb+fin+sim+opt%=க:^க #;
++verb+nonfin+sim+inf%=க:^க #;
++verb+nonfin+sim+inf%=க+negpart%=இல்லை:^கவில்லை #;
++verb+nonfin+sim+inf%=க+moodpart%=ஆம்:^கலாம் #;
++verb+nonfin+sim+inf%=க+moodpart%=அட்டும்:^கட்டும் #;
++verb+nonfin+sim+inf%=க+moodpart%=ஆகாது:^கலாகாது #;
++verb+nonfin+sim+inf%=க+sandhic%=ச்:^கச் #;
++verb+nonfin+sim+inf%=க+sandhit%=த்:^கத் #;
++verb+nonfin+sim+inf%=க+sandhik%=க்:^கக் #;
++verb+nonfin+sim+inf%=க+sandhip%=ப்:^கப் #;
++verb+nonfin+sim+verbalnoun%=தல்:^றல் #;
++verb+nonfin+sim+verbalnoun%=அது:^பது #;
++verb+nonfin+sim+verbalnoun%=அது+add%=உம்:^பதும் #;
++verb+nonfin+sim+pres%=கின்ற்+adjpart%=அ:^கின்ற #;
++verb+nonfin+sim+pres%=கிற்+adjpart%=அ:^கிற #;
++verb+nonfin+sim+futANDadjpart%=உம்:^கும் #;
+
+LEXICON NilBaseVinf
++verb+fin+sim+imp%=∅+2sg%=∅:^ #;
++verb+fin+sim+imp%=∅+2pl%=உங்கள்:^லுங்கள் #;
++verb+fin+sim+imp%=∅+neg%=ஆத்+euph%=ஏ:^லாதே #;
++verb+fin+sim+imp%=∅+neg%=ஆத்+2pl%=ஈர்கள்:^லாதீர்கள் #;
++verb+nonfin+sim+neg%=ஆ+vpart%=மல்:^லாமல் #;
++verb+nonfin+sim+neg%=ஆ+verbalnoun%=மை:^லாமை #;
++verb+nonfin+sim+neg%=ஆது+vpart%=∅:^லாது #;
++verb+nonfin+sim+neg%=ஆத்+adjpart%=அ:^லாத #;
++verb+nonfin+sim+neg%=ஆ+aux%=விடு+con%=ஆல்:^லாவிட்டால் #;
 
 LEXICON ClassKelModern
 
