--- a/tamil-part.lexc
+++ b/tamil-part.lexc
@@ -1,7 +1,7 @@
 !!!part.lexc!!!
 ! By K. Sarveswaran, iamsarves at gmail dot com.
 
-Multichar_Symbols  +noun +nom +intensifier +sandhik +sandhic +sandhip +sandhit +pron +quant +all +human +inter +poss +det +degree +same +distrib +indef +modal +neg +exist +ques +prohib +abbrev +letter +comp +acc +dat +loc +inst +gen +comparative +add +prox +dist +3sgn +cop +adjpart
+Multichar_Symbols  +noun +nom +intensifier +sandhik +sandhic +sandhip +sandhit +pron +quant +all +human +inter +poss +det +degree +same +distrib +indef +modal +neg +exist +ques +prohib +abbrev +letter +comp +acc +dat +loc +inst +gen +comparative +add +prox +dist +3sgn +cop +adjpart +1pl +3pl +adv +time
 
 LEXICON Root
 abbreviations;
@@ -30,6 +30,20 @@
 எஸ்+abbrev+letter:எஸ் #;
 
 LEXICON pronounQuantifiers
+அது+pron+dem+dist+3sgn+acc:அதனை #;
+அது+pron+dem+dist+3sgn+acc+sandhip:அதனைப் #;
+அது+pron+dem+dist+3sgn+dat:அதற்கு #;
+அது+pron+dem+dist+3sgn+loc+add:அதிலும் #;
+அது+pron+dem+dist+3sgn+nom+add:அதுவும் #;
+அவர்கள்+pron+dem+dist+3pl+nom+add:அவர்களும் #;
+யார்+pron+inter+human+acc:யாரை #;
+யார்+pron+inter+human+dat:யாருக்கு #;
+யார்+pron+inter+human+dat+add:யார்க்கும் #;
+எவர்+pron+inter+human:எவர் #;
+எவர்+pron+inter+human+add:எவரும் #;
+யாம்+pron+1pl+nom:யாம் #;
+அனைத்து+quant+all+add:அனைத்தும் #;
+எப்போது+adv+inter+time+add:எப்போதும் #;
 தங்கள்+pron+poss:தங்கள் #;
 ஒருவர்+pron+human+sg:ஒருவர் #;
 அனைவரும்+pron+human+pl+all+add:அனைவரும் #;
