--- ThamizhiVerbs-C3.lexc
+++ ThamizhiVerbs-C3.lexc
@@ -1,7 +1,7 @@
 !!!ThamizhiFST!!!
 !!!K. Sarveswaran, iamsarves@gmail.com iamsarves.com!!!
 
-Multichar_Symbols +verb %=த் % Part +PresPart +3P +2P +1P +past% %=த்% % %  +Pres +Fut +Sg +Pl +Masc +Fem +Rat +Irr +Euph +Sandhi +Neg +Comp +Imp +Inf +RelPart +Con +Causal +Temp +Neu +PartNoun % +verb% % % erbNoun +ques +worthy +foc
+Multichar_Symbols +verb %=த் % Part +PresPart +3P +2P +1P +past% %=த்% % %  +Pres +Fut +Sg +Pl +Masc +Fem +Rat +Irr +Euph +Sandhi +Neg +Comp +Imp +Inf +RelPart +Con +Causal +Temp +Neu +PartNoun % +verb% % % erbNoun +ques +worthy +foc +purpose +cop +adjpart +comp +add
 
 LEXICON Root
 
@@ -84,11 +84,13 @@
 +verb+fin+sim+strong+past%=ற்+3sghe%=ஆர்:^ன்றார் VerbQuestion;
 +verb+fin+sim+strong+past%=ற்+3sghe%=ஆர்கள்:^ன்றார்கள் VerbQuestion;
 +verb+fin+sim+strong+past%=ற்+3sgn%=அது:^ன்றது VerbQuestion;
++verb+nonfin+sim+strong+past%=ற்+verbalnoun%=அது+comp%=ஆவது:^ன்றதாவது #;
 +verb+nonfin+sim+strong+past%=ற்+verbalnoun%=அது+acc%=ஐ:^ன்றதை #;
 +verb+nonfin+sim+strong+past%=ற்+verbalnoun%=அது+acc%=ஐ+add%=உம்:^ன்றதையும் #;
 +verb+nonfin+sim+strong+past%=ற்+verbalnoun%=அது+inst%=ஆல்:^ன்றதால் #;
 +verb+nonfin+sim+strong+past%=ற்+verbalnoun%=அது+inst%=ஆல்+add%=உம்:^ன்றதாலும் #;
 +verb+nonfin+sim+strong+past%=ற்+verbalnoun%=அது+dat%=க்கு:^ன்றதற்கு #;
++verb+nonfin+sim+strong+past%=ற்+verbalnoun%=அது+dat%=க்கு+purpose+cop%=ஆ+adjpart%=ன:^ன்றதற்கான #;
 +verb+nonfin+sim+strong+past%=ற்+verbalnoun%=அது+dat%=க்கு+add%=உம்:^ன்றதற்கும் #;
 +verb+nonfin+sim+strong+past%=ற்+verbalnoun%=அது+loc%=இல்:^ன்றதில் #;
 +verb+nonfin+sim+strong+past%=ற்+verbalnoun%=அது+loc%=இல்+add%=உம்:^ன்றதிலும் #;
@@ -122,11 +124,13 @@
 +verb+fin+sim+strong+pres%=கிற்+3sghe%=ஆர்:^கிறார் VerbQuestion;
 +verb+fin+sim+strong+pres%=கிற்+3sghe%=ஆர்கள்:^கிறார்கள் VerbQuestion;
 +verb+fin+sim+strong+pres%=கிற்+3sgn%=அது:^கிறது VerbQuestion;
++verb+nonfin+sim+strong+pres%=கிற்+verbalnoun%=அது+comp%=ஆவது:^கிறதாவது #;
 +verb+nonfin+sim+strong+pres%=கிற்+verbalnoun%=அது+acc%=ஐ:^கிறதை #;
 +verb+nonfin+sim+strong+pres%=கிற்+verbalnoun%=அது+acc%=ஐ+add%=உம்:^கிறதையும் #;
 +verb+nonfin+sim+strong+pres%=கிற்+verbalnoun%=அது+inst%=ஆல்:^கிறதால் #;
 +verb+nonfin+sim+strong+pres%=கிற்+verbalnoun%=அது+inst%=ஆல்+add%=உம்:^கிறதாலும் #;
 +verb+nonfin+sim+strong+pres%=கிற்+verbalnoun%=அது+dat%=க்கு:^கிறதற்கு #;
++verb+nonfin+sim+strong+pres%=கிற்+verbalnoun%=அது+dat%=க்கு+purpose+cop%=ஆ+adjpart%=ன:^கிறதற்கான #;
 +verb+nonfin+sim+strong+pres%=கிற்+verbalnoun%=அது+dat%=க்கு+add%=உம்:^கிறதற்கும் #;
 +verb+nonfin+sim+strong+pres%=கிற்+verbalnoun%=அது+loc%=இல்:^கிறதில் #;
 +verb+nonfin+sim+strong+pres%=கிற்+verbalnoun%=அது+loc%=இல்+add%=உம்:^கிறதிலும் #;
@@ -160,11 +164,13 @@
 +verb+fin+sim+strong+pres%=கின்ற்+3ple%=ஆர்கள்:^கின்றார்கள் VerbQuestion;
 +verb+fin+sim+strong+pres%=கின்ற்+3sghe%=ஆர்கள்:^கின்றார்கள் VerbQuestion;
 +verb+fin+sim+strong+pres%=கின்ற்+3sgn%=அது:^கின்றது VerbQuestion;
++verb+nonfin+sim+strong+pres%=கின்ற்+verbalnoun%=அது+comp%=ஆவது:^கின்றதாவது #;
 +verb+nonfin+sim+strong+pres%=கின்ற்+verbalnoun%=அது+acc%=ஐ:^கின்றதை #;
 +verb+nonfin+sim+strong+pres%=கின்ற்+verbalnoun%=அது+acc%=ஐ+add%=உம்:^கின்றதையும் #;
 +verb+nonfin+sim+strong+pres%=கின்ற்+verbalnoun%=அது+inst%=ஆல்:^கின்றதால் #;
 +verb+nonfin+sim+strong+pres%=கின்ற்+verbalnoun%=அது+inst%=ஆல்+add%=உம்:^கின்றதாலும் #;
 +verb+nonfin+sim+strong+pres%=கின்ற்+verbalnoun%=அது+dat%=க்கு:^கின்றதற்கு #;
++verb+nonfin+sim+strong+pres%=கின்ற்+verbalnoun%=அது+dat%=க்கு+purpose+cop%=ஆ+adjpart%=ன:^கின்றதற்கான #;
 +verb+nonfin+sim+strong+pres%=கின்ற்+verbalnoun%=அது+dat%=க்கு+add%=உம்:^கின்றதற்கும் #;
 +verb+nonfin+sim+strong+pres%=கின்ற்+verbalnoun%=அது+loc%=இல்:^கின்றதில் #;
 +verb+nonfin+sim+strong+pres%=கின்ற்+verbalnoun%=அது+loc%=இல்+add%=உம்:^கின்றதிலும் #;
@@ -207,11 +213,13 @@
 +verb+fin+sim+strong+pres%=கின்ற்+euph%=அன்+3sgm%=அன்:^கின்றனன் VerbQuestion;
 +verb+fin+sim+strong+pres%=கின்ற்+euph%=அன்+3sgf%=அள்:^கின்றனள் VerbQuestion;
 +verb+nonfin+sim+verbalnoun%=அது:^வது #;
++verb+nonfin+sim+verbalnoun%=அது+comp%=ஆவது:^வதாவது #;
 +verb+nonfin+sim+verbalnoun%=அது+acc%=ஐ:^வதை #;
 +verb+nonfin+sim+verbalnoun%=அது+acc%=ஐ+add%=உம்:^வதையும் #;
 +verb+nonfin+sim+verbalnoun%=அது+inst%=ஆல்:^வதால் #;
 +verb+nonfin+sim+verbalnoun%=அது+inst%=ஆல்+add%=உம்:^வதாலும் #;
 +verb+nonfin+sim+verbalnoun%=அது+dat%=க்கு:^வதற்கு #;
++verb+nonfin+sim+verbalnoun%=அது+dat%=க்கு+purpose+cop%=ஆ+adjpart%=ன:^வதற்கான #;
 +verb+nonfin+sim+verbalnoun%=அது+dat%=க்கு+add%=உம்:^வதற்கும் #;
 +verb+nonfin+sim+verbalnoun%=அது+loc%=இல்:^வதில் #;
 +verb+nonfin+sim+verbalnoun%=அது+loc%=இல்+add%=உம்:^வதிலும் #;
@@ -250,11 +258,13 @@
 +verb+fin+sim+caus%=வி+past%=ற்+3ple%=ஆர்கள்:^வின்றார்கள் VerbQuestion;
 +verb+fin+sim+caus%=வி+past%=ற்+3sghe%=ஆர்கள்:^வின்றார்கள் VerbQuestion;
 +verb+fin+sim+caus%=வி+past%=ற்+3sgn%=அது:^வின்றது VerbQuestion;
++verb+nonfin+sim+caus%=வி+past%=ற்+verbalnoun%=அது+comp%=ஆவது:^வின்றதாவது #;
 +verb+nonfin+sim+caus%=வி+past%=ற்+verbalnoun%=அது+acc%=ஐ:^வின்றதை #;
 +verb+nonfin+sim+caus%=வி+past%=ற்+verbalnoun%=அது+acc%=ஐ+add%=உம்:^வின்றதையும் #;
 +verb+nonfin+sim+caus%=வி+past%=ற்+verbalnoun%=அது+inst%=ஆல்:^வின்றதால் #;
 +verb+nonfin+sim+caus%=வி+past%=ற்+verbalnoun%=அது+inst%=ஆல்+add%=உம்:^வின்றதாலும் #;
 +verb+nonfin+sim+caus%=வி+past%=ற்+verbalnoun%=அது+dat%=க்கு:^வின்றதற்கு #;
++verb+nonfin+sim+caus%=வி+past%=ற்+verbalnoun%=அது+dat%=க்கு+purpose+cop%=ஆ+adjpart%=ன:^வின்றதற்கான #;
 +verb+nonfin+sim+caus%=வி+past%=ற்+verbalnoun%=அது+dat%=க்கு+add%=உம்:^வின்றதற்கும் #;
 +verb+nonfin+sim+caus%=வி+past%=ற்+verbalnoun%=அது+loc%=இல்:^வின்றதில் #;
 +verb+nonfin+sim+caus%=வி+past%=ற்+verbalnoun%=அது+loc%=இல்+add%=உம்:^வின்றதிலும் #;
@@ -287,11 +297,13 @@
 +verb+fin+sim+caus%=வி+pres%=கிற்+3ple%=ஆர்கள்:^விகிறார்கள் VerbQuestion;
 +verb+fin+sim+caus%=வி+pres%=கிற்+3sghe%=ஆர்கள்:^விகிறார்கள் VerbQuestion;
 +verb+fin+sim+caus%=வி+pres%=கிற்+3sgn%=அது:^விகிறது VerbQuestion;
++verb+nonfin+sim+caus%=வி+pres%=கிற்+verbalnoun%=அது+comp%=ஆவது:^விகிறதாவது #;
 +verb+nonfin+sim+caus%=வி+pres%=கிற்+verbalnoun%=அது+acc%=ஐ:^விகிறதை #;
 +verb+nonfin+sim+caus%=வி+pres%=கிற்+verbalnoun%=அது+acc%=ஐ+add%=உம்:^விகிறதையும் #;
 +verb+nonfin+sim+caus%=வி+pres%=கிற்+verbalnoun%=அது+inst%=ஆல்:^விகிறதால் #;
 +verb+nonfin+sim+caus%=வி+pres%=கிற்+verbalnoun%=அது+inst%=ஆல்+add%=உம்:^விகிறதாலும் #;
 +verb+nonfin+sim+caus%=வி+pres%=கிற்+verbalnoun%=அது+dat%=க்கு:^விகிறதற்கு #;
++verb+nonfin+sim+caus%=வி+pres%=கிற்+verbalnoun%=அது+dat%=க்கு+purpose+cop%=ஆ+adjpart%=ன:^விகிறதற்கான #;
 +verb+nonfin+sim+caus%=வி+pres%=கிற்+verbalnoun%=அது+dat%=க்கு+add%=உம்:^விகிறதற்கும் #;
 +verb+nonfin+sim+caus%=வி+pres%=கிற்+verbalnoun%=அது+loc%=இல்:^விகிறதில் #;
 +verb+nonfin+sim+caus%=வி+pres%=கிற்+verbalnoun%=அது+loc%=இல்+add%=உம்:^விகிறதிலும் #;
@@ -324,11 +336,13 @@
 +verb+fin+sim+caus%=வி+pres%=கின்ற்+3ple%=ஆர்கள்:^விகின்றார்கள் VerbQuestion;
 +verb+fin+sim+caus%=வி+pres%=கின்ற்+3sghe%=ஆர்கள்:^விகின்றார்கள் VerbQuestion;
 +verb+fin+sim+caus%=வி+pres%=கின்ற்+3sgn%=அது:^விகின்றது VerbQuestion;
++verb+nonfin+sim+caus%=வி+pres%=கின்ற்+verbalnoun%=அது+comp%=ஆவது:^விகின்றதாவது #;
 +verb+nonfin+sim+caus%=வி+pres%=கின்ற்+verbalnoun%=அது+acc%=ஐ:^விகின்றதை #;
 +verb+nonfin+sim+caus%=வி+pres%=கின்ற்+verbalnoun%=அது+acc%=ஐ+add%=உம்:^விகின்றதையும் #;
 +verb+nonfin+sim+caus%=வி+pres%=கின்ற்+verbalnoun%=அது+inst%=ஆல்:^விகின்றதால் #;
 +verb+nonfin+sim+caus%=வி+pres%=கின்ற்+verbalnoun%=அது+inst%=ஆல்+add%=உம்:^விகின்றதாலும் #;
 +verb+nonfin+sim+caus%=வி+pres%=கின்ற்+verbalnoun%=அது+dat%=க்கு:^விகின்றதற்கு #;
++verb+nonfin+sim+caus%=வி+pres%=கின்ற்+verbalnoun%=அது+dat%=க்கு+purpose+cop%=ஆ+adjpart%=ன:^விகின்றதற்கான #;
 +verb+nonfin+sim+caus%=வி+pres%=கின்ற்+verbalnoun%=அது+dat%=க்கு+add%=உம்:^விகின்றதற்கும் #;
 +verb+nonfin+sim+caus%=வி+pres%=கின்ற்+verbalnoun%=அது+loc%=இல்:^விகின்றதில் #;
 +verb+nonfin+sim+caus%=வி+pres%=கின்ற்+verbalnoun%=அது+loc%=இல்+add%=உம்:^விகின்றதிலும் #;
@@ -383,11 +397,17 @@
 +verb+nonfin+sim+inf%=அ+moodpart%=ஆகாது+sandhit%=த்:^அலாகாதுத் #;
 +verb+nonfin+sim+inf%=அ+moodpart%=ஆகாது+sandhik%=க்:^அலாகாதுக் #;
 +verb+nonfin+sim+inf%=அ:^அ #;
++verb+nonfin+sim+inf%=அ+add%=உம்:^அவும் #;
 +verb+nonfin+sim+inf%=அ+sandhic%=ச்:^அச் #;
++verb+nonfin+sim+inf%=அ+sandhic%=ச்+add%=உம்:^அச்வும் #;
 +verb+nonfin+sim+inf%=அ+sandhit%=த்:^அத் #;
++verb+nonfin+sim+inf%=அ+sandhit%=த்+add%=உம்:^அத்வும் #;
 +verb+nonfin+sim+inf%=அ+sandhik%=க்:^அக் #;
++verb+nonfin+sim+inf%=அ+sandhik%=க்+add%=உம்:^அக்வும் #;
 +verb+nonfin+sim+inf%=அ+sandhip%=ப்:^அப் #;
++verb+nonfin+sim+inf%=அ+sandhip%=ப்+add%=உம்:^அப்வும் #;
 +verb+nonfin+sim+vpart%=உ:^து #;
++verb+nonfin+sim+vpart%=உ+add%=உம்:^தும் #;
 +verb+nonfin+sim+vpart%=உ+foc%=ஏ:^தே #;
 +verb+nonfin+sim+vpart%=உ+sandhic%=ச்:^துச் #;
 +verb+nonfin+sim+vpart%=உ+sandhit%=த்:^துத் #;
@@ -400,12 +420,15 @@
 +verb+nonfin+sim+neg%=ஆ+vpart%=மல்+sandhik%=க்:^ாமல்க் #;
 +verb+nonfin+sim+neg%=ஆ+verbalnoun%=மை:^ாமை #;
 +verb+nonfin+sim+neg%=ஆது+vpart%=∅:^ாது #;
++verb+nonfin+sim+neg%=ஆது+vpart%=∅+add%=உம்:^ாதும் #;
 +verb+nonfin+sim+neg%=ஆது+vpart%=∅+sandhip%=ப்:^ாதுப் #;
 +verb+nonfin+sim+neg%=ஆது+vpart%=∅+sandhic%=ச்:^ாதுச் #;
 +verb+nonfin+sim+neg%=ஆது+vpart%=∅+sandhit%=த்:^ாதுத் #;
 +verb+nonfin+sim+neg%=ஆது+vpart%=∅+sandhik%=க்:^ாதுக் #;
 +verb+nonfin+sim+con%=ஆல்:^தால் #;
++verb+nonfin+sim+con%=ஆல்+add%=உம்:^தாலும் #;
 +verb+nonfin+sim+neg%=ஆ+aux%=விடு+con%=ஆல்:^ாவிட்டால் #;
++verb+nonfin+sim+neg%=ஆ+aux%=விடு+con%=ஆல்+add%=உம்:^ாவிட்டாலும் #;
 +verb+nonfin+sim+past%=ற்+adjpart%=அ:^ன்ற #;
 +verb+nonfin+sim+pres%=கின்ற்+adjpart%=அ:^கின்ற #;
 +verb+nonfin+sim+pres%=கிற்+adjpart%=அ:^கிற #;
@@ -428,13 +451,16 @@
 +verb+fin+complex+perfect+pres%=∅+3ple%=னர்:^ன்றுள்ளனர் VerbQuestion;
 +verb+fin+complex+perfect+pres%=∅+3ple%=ஆர்கள்:^ன்றுள்ளார்கள் VerbQuestion;
 +verb+fin+complex+perfect+pres%=∅+3sgn%=அது:^ன்றுள்ளது VerbQuestion;
++verb+nonfin+complex+perfect+pres%=∅+verbalnoun%=அது+comp%=ஆவது:^ன்றுள்ளதாவது #;
 +verb+fin+complex+perfect+pres%=∅+3pln%=அன:^ன்றுள்ளன VerbQuestionNeuterPlural;
 +verb+nonfin+complex+perfect+pres%=∅+verbalnoun%=அது:^ன்றுள்ளது #;
++verb+nonfin+complex+perfect+pres%=∅+verbalnoun%=அது+comp%=ஆவது:^ன்றுள்ளதாவது #;
 +verb+nonfin+complex+perfect+pres%=∅+verbalnoun%=அது+acc%=ஐ:^ன்றுள்ளதை #;
 +verb+nonfin+complex+perfect+pres%=∅+verbalnoun%=அது+acc%=ஐ+add%=உம்:^ன்றுள்ளதையும் #;
 +verb+nonfin+complex+perfect+pres%=∅+verbalnoun%=அது+inst%=ஆல்:^ன்றுள்ளதால் #;
 +verb+nonfin+complex+perfect+pres%=∅+verbalnoun%=அது+inst%=ஆல்+add%=உம்:^ன்றுள்ளதாலும் #;
 +verb+nonfin+complex+perfect+pres%=∅+verbalnoun%=அது+dat%=க்கு:^ன்றுள்ளதற்கு #;
++verb+nonfin+complex+perfect+pres%=∅+verbalnoun%=அது+dat%=க்கு+purpose+cop%=ஆ+adjpart%=ன:^ன்றுள்ளதற்கான #;
 +verb+nonfin+complex+perfect+pres%=∅+verbalnoun%=அது+dat%=க்கு+add%=உம்:^ன்றுள்ளதற்கும் #;
 +verb+nonfin+complex+perfect+pres%=∅+verbalnoun%=அது+loc%=இல்:^ன்றுள்ளதில் #;
 +verb+nonfin+complex+perfect+pres%=∅+verbalnoun%=அது+loc%=இல்+add%=உம்:^ன்றுள்ளதிலும் #;
@@ -598,6 +624,7 @@
 +verb+nonfin+sim+weak+fut%=வ்+PartNoun%=அவர்+human+pl+soc:^பவர்களுடன் #;
 +verb+nonfin+sim+weak+fut%=வ்+PartNoun%=அவர்+human+pl+soc+add:^பவர்களுடனும் #;
 +verb+nonfin+sim+weak+fut%=வ்+verbalnoun%=அது:^வது #;
++verb+nonfin+sim+weak+fut%=வ்+verbalnoun%=அது+comp%=ஆவது:^வதாவது #;
 +verb+nonfin+sim+weak+fut%=வ்+verbalnoun%=அது+add%=உம்:^வதும் #;
 +verb+nonfin+sim+weak+fut%=வ்+verbalnoun%=அது+foc%=ஏ:^வதே #;
 +verb+nonfin+sim+weak+fut%=வ்+verbalnoun%=அது+foc%=தான்:^வதுதான் #;
@@ -606,6 +633,7 @@
 +verb+nonfin+sim+weak+fut%=வ்+verbalnoun%=அது+inst%=ஆல்:^வதால் #;
 +verb+nonfin+sim+weak+fut%=வ்+verbalnoun%=அது+inst%=ஆல்+add%=உம்:^வதாலும் #;
 +verb+nonfin+sim+weak+fut%=வ்+verbalnoun%=அது+dat%=க்கு:^வதற்கு #;
++verb+nonfin+sim+weak+fut%=வ்+verbalnoun%=அது+dat%=க்கு+purpose+cop%=ஆ+adjpart%=ன:^வதற்கான #;
 +verb+nonfin+sim+weak+fut%=வ்+verbalnoun%=அது+dat%=க்கு+add%=உம்:^வதற்கும் #;
 +verb+nonfin+sim+weak+fut%=வ்+verbalnoun%=அது+loc%=இல்:^வதில் #;
 +verb+nonfin+sim+weak+fut%=வ்+verbalnoun%=அது+loc%=இல்+add%=உம்:^வதிலும் #;
