--- ThamizhiVerbs-C12.lexc
+++ ThamizhiVerbs-C12.lexc
@@ -1,7 +1,7 @@
 !!!ThamizhiFST!!!
 !!!K. Sarveswaran, iamsarves@gmail.com iamsarves.com!!!
 
-Multichar_Symbols +verb %=த் % Part +PresPart +3P +2P +1P +past% %=த்% % %  +Pres +Fut +Sg +Pl +Masc +Fem +Rat +Irr +Euph +Sandhi +Neg +Comp +Imp +Inf +RelPart +Con +Causal +Temp +Neu +PartNoun % +verb% % % erbNoun +ques +worthy +foc +purpose +cop +adjpart +comp +add +vpart_tum +vpart_potu +cond +vpart_maaru +exist
+Multichar_Symbols +verb %=த் % Part +PresPart +3P +2P +1P +past% %=த்% % %  +Pres +Fut +Sg +Pl +Masc +Fem +Rat +Irr +Euph +Sandhi +Neg +Comp +Imp +Inf +RelPart +Con +Causal +Temp +Neu +PartNoun % +verb% % % erbNoun +ques +worthy +foc +purpose +cop +adjpart +comp +add +vpart_tum +vpart_potu +cond +vpart_maaru +exist +psp_padi +vpart_pin
 
 
 LEXICON Root
@@ -249,6 +249,8 @@
 
 ! Productive infinitive + தக்க suitability/worthiness construction.
 +verb+nonfin+sim+inf%=க்க+worthy%=தக்க+adjpart%=அ:^க்கத்தக்க #;
++verb+nonfin+sim+inf%=க்க+worthy%=தக்க+adjpart%=அ+psp_padi%=படி:^க்கத்தக்கபடி #;
++verb+nonfin+sim+inf%=க்க+worthy%=தக்க+adjpart%=அ+vpart_pin%=பின்:^க்கத்தக்கபின் #;
 +verb+nonfin+sim+cond%=இன்:^ப்பின் #;
 +verb+nonfin+sim+cond%=இன்+add%=உம்:^ப்பினும் #;
 +verb+nonfin+sim+inf%=க்க+worthy%=தக்க+adjpart%=அ+vpart_potu%=போது:^க்கத்தக்கபோது #;
@@ -664,16 +666,26 @@
 +verb+nonfin+sim+neg%=ஆ+aux%=விடு+con%=ஆல்:^க்காவிட்டால் #;
 +verb+nonfin+sim+neg%=ஆ+aux%=விடு+con%=ஆல்+add%=உம்:^க்காவிட்டாலும் #;
 +verb+nonfin+sim+past%=த்+adjpart%=அ:^ந்த #;
++verb+nonfin+sim+past%=த்+adjpart%=அ+psp_padi%=படி:^ந்தபடி #;
++verb+nonfin+sim+past%=த்+adjpart%=அ+vpart_pin%=பின்:^ந்தபின் #;
 +verb+nonfin+sim+past%=த்+adjpart%=அ+vpart_potu%=போது:^ந்தபோது #;
 +verb+nonfin+sim+past%=த்+adjpart%=அ+PartNoun%=ஓர்:^ந்தோர் #;
 +verb+nonfin+sim+pres%=க்கின்ற்+adjpart%=அ:^க்கின்ற #;
++verb+nonfin+sim+pres%=க்கின்ற்+adjpart%=அ+psp_padi%=படி:^க்கின்றபடி #;
++verb+nonfin+sim+pres%=க்கின்ற்+adjpart%=அ+vpart_pin%=பின்:^க்கின்றபின் #;
 +verb+nonfin+sim+pres%=க்கின்ற்+adjpart%=அ+vpart_potu%=போது:^க்கின்றபோது #;
 +verb+nonfin+sim+pres%=க்கிற்+adjpart%=அ:^க்கிற #;
++verb+nonfin+sim+pres%=க்கிற்+adjpart%=அ+psp_padi%=படி:^க்கிறபடி #;
++verb+nonfin+sim+pres%=க்கிற்+adjpart%=அ+vpart_pin%=பின்:^க்கிறபின் #;
 +verb+nonfin+sim+pres%=க்கிற்+adjpart%=அ+vpart_potu%=போது:^க்கிறபோது #;
 +verb+nonfin+sim+futANDadjpart%=உம்:^க்கும் #;
++verb+nonfin+sim+futANDadjpart%=உம்+psp_padi%=படி:^க்கும்படி #;
++verb+nonfin+sim+futANDadjpart%=உம்+vpart_pin%=பின்:^க்கும்பின் #;
 +verb+nonfin+sim+futANDadjpart%=உம்+vpart_maaru%=மாறு:^க்குமாறு #;
 +verb+nonfin+sim+futANDadjpart%=உம்+vpart_potu%=போது:^க்கும்போது #;
 +verb+nonfin+sim+neg%=ஆத்+adjpart%=அ:^ாத #;
++verb+nonfin+sim+neg%=ஆத்+adjpart%=அ+psp_padi%=படி:^ாதபடி #;
++verb+nonfin+sim+neg%=ஆத்+adjpart%=அ+vpart_pin%=பின்:^ாதபின் #;
 +verb+nonfin+sim+neg%=ஆத்+adjpart%=அ+vpart_potu%=போது:^ாதபோது #;
 
 
@@ -723,6 +735,8 @@
 +verb+nonfin+complex+perfect+pres%=∅+verbalnoun%=அது+trans%=ஆக+cop+fut+3sgn:^ந்துள்ளதாகும் #;
 +verb+nonfin+complex+perfect+pres%=∅+verbalnoun%=அது+trans%=ஆக+add%=உம்:^ந்துள்ளதாகவும் #;
 +verb+nonfin+complex+perfect+pres%=∅+adjpart%=அ:^ந்துள்ள #;
++verb+nonfin+complex+perfect+pres%=∅+adjpart%=அ+psp_padi%=படி:^ந்துள்ளபடி #;
++verb+nonfin+complex+perfect+pres%=∅+adjpart%=அ+vpart_pin%=பின்:^ந்துள்ளபின் #;
 +verb+nonfin+complex+perfect+pres%=∅+adjpart%=அ+vpart_potu%=போது:^ந்துள்ளபோது #;
 
 ! Productive participial human and future-neuter nominals.
@@ -4416,14 +4430,24 @@
 +verb+nonfin+complex+aspect+neg%=ஆ+aux%=விடு+con%=ஆல்:^க்காவிட்டால் #;
 +verb+nonfin+complex+aspect+neg%=ஆ+aux%=விடு+con%=ஆல்+add%=உம்:^க்காவிட்டாலும் #;
 +verb+nonfin+complex+aspect+past%=த்+adjpart%=அ:^ந்த #;
++verb+nonfin+complex+aspect+past%=த்+adjpart%=அ+psp_padi%=படி:^ந்தபடி #;
++verb+nonfin+complex+aspect+past%=த்+adjpart%=அ+vpart_pin%=பின்:^ந்தபின் #;
 +verb+nonfin+complex+aspect+past%=த்+adjpart%=அ+vpart_potu%=போது:^ந்தபோது #;
 +verb+nonfin+complex+aspect+pres%=க்கின்ற்+adjpart%=அ:^க்கின்ற #;
++verb+nonfin+complex+aspect+pres%=க்கின்ற்+adjpart%=அ+psp_padi%=படி:^க்கின்றபடி #;
++verb+nonfin+complex+aspect+pres%=க்கின்ற்+adjpart%=அ+vpart_pin%=பின்:^க்கின்றபின் #;
 +verb+nonfin+complex+aspect+pres%=க்கின்ற்+adjpart%=அ+vpart_potu%=போது:^க்கின்றபோது #;
 +verb+nonfin+complex+aspect+pres%=க்கிற்+adjpart%=அ:^க்கிற #;
++verb+nonfin+complex+aspect+pres%=க்கிற்+adjpart%=அ+psp_padi%=படி:^க்கிறபடி #;
++verb+nonfin+complex+aspect+pres%=க்கிற்+adjpart%=அ+vpart_pin%=பின்:^க்கிறபின் #;
 +verb+nonfin+complex+aspect+pres%=க்கிற்+adjpart%=அ+vpart_potu%=போது:^க்கிறபோது #;
 +verb+nonfin+complex+aspect+futANDadjpart%=உம்:^க்கும் #;
++verb+nonfin+complex+aspect+futANDadjpart%=உம்+psp_padi%=படி:^க்கும்படி #;
++verb+nonfin+complex+aspect+futANDadjpart%=உம்+vpart_pin%=பின்:^க்கும்பின் #;
 +verb+nonfin+complex+aspect+futANDadjpart%=உம்+vpart_potu%=போது:^க்கும்போது #;
 +verb+nonfin+complex+aspect+neg%=ஆத்+adjpart%=அ:^ாத #;
++verb+nonfin+complex+aspect+neg%=ஆத்+adjpart%=அ+psp_padi%=படி:^ாதபடி #;
++verb+nonfin+complex+aspect+neg%=ஆத்+adjpart%=அ+vpart_pin%=பின்:^ாதபின் #;
 +verb+nonfin+complex+aspect+neg%=ஆத்+adjpart%=அ+vpart_potu%=போது:^ாதபோது #;
 
 
@@ -8086,14 +8110,24 @@
 +verb+nonfin+complex+aspect+neg%=ஆ+aux%=விடு+con%=ஆல்:^க்காவிட்டால் #;
 +verb+nonfin+complex+aspect+neg%=ஆ+aux%=விடு+con%=ஆல்+add%=உம்:^க்காவிட்டாலும் #;
 +verb+nonfin+complex+aspect+past%=த்+adjpart%=அ:^ந்த #;
++verb+nonfin+complex+aspect+past%=த்+adjpart%=அ+psp_padi%=படி:^ந்தபடி #;
++verb+nonfin+complex+aspect+past%=த்+adjpart%=அ+vpart_pin%=பின்:^ந்தபின் #;
 +verb+nonfin+complex+aspect+past%=த்+adjpart%=அ+vpart_potu%=போது:^ந்தபோது #;
 +verb+nonfin+complex+aspect+pres%=க்கின்ற்+adjpart%=அ:^க்கின்ற #;
++verb+nonfin+complex+aspect+pres%=க்கின்ற்+adjpart%=அ+psp_padi%=படி:^க்கின்றபடி #;
++verb+nonfin+complex+aspect+pres%=க்கின்ற்+adjpart%=அ+vpart_pin%=பின்:^க்கின்றபின் #;
 +verb+nonfin+complex+aspect+pres%=க்கின்ற்+adjpart%=அ+vpart_potu%=போது:^க்கின்றபோது #;
 +verb+nonfin+complex+aspect+pres%=க்கிற்+adjpart%=அ:^க்கிற #;
++verb+nonfin+complex+aspect+pres%=க்கிற்+adjpart%=அ+psp_padi%=படி:^க்கிறபடி #;
++verb+nonfin+complex+aspect+pres%=க்கிற்+adjpart%=அ+vpart_pin%=பின்:^க்கிறபின் #;
 +verb+nonfin+complex+aspect+pres%=க்கிற்+adjpart%=அ+vpart_potu%=போது:^க்கிறபோது #;
 +verb+nonfin+complex+aspect+futANDadjpart%=உம்:^க்கும் #;
++verb+nonfin+complex+aspect+futANDadjpart%=உம்+psp_padi%=படி:^க்கும்படி #;
++verb+nonfin+complex+aspect+futANDadjpart%=உம்+vpart_pin%=பின்:^க்கும்பின் #;
 +verb+nonfin+complex+aspect+futANDadjpart%=உம்+vpart_potu%=போது:^க்கும்போது #;
 +verb+nonfin+complex+aspect+neg%=ஆத்+adjpart%=அ:^ாத #;
++verb+nonfin+complex+aspect+neg%=ஆத்+adjpart%=அ+psp_padi%=படி:^ாதபடி #;
++verb+nonfin+complex+aspect+neg%=ஆத்+adjpart%=அ+vpart_pin%=பின்:^ாதபின் #;
 +verb+nonfin+complex+aspect+neg%=ஆத்+adjpart%=அ+vpart_potu%=போது:^ாதபோது #;
 
 
