--- ThamizhiVerbs-otherthan-3-4-62-11-12.lexc
+++ ThamizhiVerbs-otherthan-3-4-62-11-12.lexc
@@ -802,6 +802,7 @@
 +verb+fin+sim+caus%=வி+fut%=ப்ப்+3pln%=∅:^விக்கும் VerbQuestion;
 +verb+fin+sim+opt%=க:^க #;
 +verb+nonfin+sim+inf%=அ+negpart%=இல்லை:^அவில்லை #;
++verb+nonfin+sim+inf%=அ+negpart%=இல்லை+colloq:^அல #;
 +verb+nonfin+sim+inf%=அ+negpart%=இல்லை+sandhip%=ப்:^அவில்லைப் #;
 +verb+nonfin+sim+inf%=அ+negpart%=இல்லை+sandhic%=ச்:^அவில்லைச் #;
 +verb+nonfin+sim+inf%=அ+negpart%=இல்லை+sandhit%=த்:^அவில்லைத் #;
@@ -10023,6 +10024,7 @@
 +verb+nonfin+sim+strong+past%=இன்+verbalnoun%=அது+trans%=ஆக+add%=உம்:^னதாகவும் #;
 +verb+fin+sim+strong+fut%=உம்+3sgn%=∅:^கும் VerbQuestion;
 +verb+nonfin+sim+inf%=க+negpart%=இல்லை:^கவில்லை #;
++verb+nonfin+sim+inf%=க+negpart%=இல்லை+colloq:^கல #;
 +verb+nonfin+sim+inf%=க+negpart%=இல்லை+sandhip%=ப்:^கவில்லைப் #;
 +verb+nonfin+sim+inf%=க+negpart%=இல்லை+sandhic%=ச்:^கவில்லைச் #;
 +verb+nonfin+sim+inf%=க+negpart%=இல்லை+sandhit%=த்:^கவில்லைத் #;
@@ -14778,6 +14780,7 @@
 +verb+fin+sim+middle+fut%=ப்+3sghe%=ஆர்கள்:^்பார்கள் VerbQuestion;
 +verb+fin+sim+strong+fut%=உம்+3sgn%=∅:^்கும் VerbQuestion;
 +verb+nonfin+sim+inf%=அ+negpart%=இல்லை:^்கவில்லை #;
++verb+nonfin+sim+inf%=அ+negpart%=இல்லை+colloq:^்கல #;
 +verb+nonfin+sim+inf%=அ+negpart%=இல்லை+sandhip%=ப்:^்கவில்லைப் #;
 +verb+nonfin+sim+inf%=அ+negpart%=இல்லை+sandhic%=ச்:^்கவில்லைச் #;
 +verb+nonfin+sim+inf%=அ+negpart%=இல்லை+sandhit%=த்:^்கவில்லைத் #;
@@ -20775,6 +20778,7 @@
 +verb+fin+sim+caus%=வி+fut%=ப்ப்+3sgn%=∅:^க்கும் VerbQuestion;
 +verb+fin+sim+opt%=க:^க #;
 +verb+nonfin+sim+inf%=அ+negpart%=இல்லை:^கவில்லை #;
++verb+nonfin+sim+inf%=அ+negpart%=இல்லை+colloq:^கல #;
 +verb+nonfin+sim+inf%=அ+negpart%=இல்லை+sandhip%=ப்:^கவில்லைப் #;
 +verb+nonfin+sim+inf%=அ+negpart%=இல்லை+sandhic%=ச்:^கவில்லைச் #;
 +verb+nonfin+sim+inf%=அ+negpart%=இல்லை+sandhit%=த்:^கவில்லைத் #;
