--- ThamizhiVerbs-C12.lexc
+++ ThamizhiVerbs-C12.lexc
@@ -267,6 +267,7 @@
 
 ! Productive infinitive negation and possibility missing from Class 12.
 +verb+nonfin+sim+inf%=க்க+negpart%=இல்லை:^க்கவில்லை #;
++verb+nonfin+sim+inf%=க்க+negpart%=இல்லை+colloq:^க்கல #;
 +verb+nonfin+sim+inf%=க்க+moodpart%=ஆம்:^க்கலாம் VerbQuestion;
 
 ! Reviewed dictionary citation continuation
@@ -4188,6 +4189,7 @@
 
 ! Productive infinitive negation and possibility missing from Class 12.
 +verb+nonfin+complex+aspect+inf%=க்க+negpart%=இல்லை:^க்கவில்லை #;
++verb+nonfin+complex+aspect+inf%=க்க+negpart%=இல்லை+colloq:^க்கல #;
 +verb+nonfin+complex+aspect+inf%=க்க+moodpart%=ஆம்:^க்கலாம் VerbQuestion;
 
 +verb+fin+complex+aspect+strong+past%=த்+1sg%=ஏன்:^ந்தேன் VerbQuestion;
